src/styles/blocks/header.scss

```diff
@@ -0,0 +1,12 @@
+.header {
+  @extend %flex-space-between;
+  position: sticky;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    display: flex;
+    align-items: center;
+    margin-left: 50px;
```

Comment:
it's a distance between block border and it's content, so better make it a padding for your header from both sides

Reply:
Agree with Illia, fix this